### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by amaredeus**
## 1.0.0-BETA (2021-12-14)

1. Add initial nextjs setup
2. Integration discord auth
   - integrate twitter auth
   - link accounts
3. Adjust to client side verification
4. Add twitter claim page

### CODE REVIEW

File: src/pages/verification/[platform].tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import {
+	GetServerSidePropsContext, GetServerSidePropsResult,
```
Comment (nonsensecodes) [inline src/pages/verification/[platform].tsx:2]: Will you place `GetServerSidePropsResult` on its own line, for consistency?

File: src/pages/claim/[code].tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+        const twitterAuth: TwitterAuthentication = await TwitterAuth.authLink();
+        return {
+            redirect: {
+                destination: twitterAuth.url,
+                permanent: false,
+            },
+        };
+    }
+    // const res = await fetch(`/api/poap/tweet?code=${claimCode}`);
```
Comment (nonsensecodes) [inline src/pages/claim/[code].tsx:87]: Is this intentionally left here to return to at a later time?

**Review by nonsensecodes — CHANGES_REQUESTED** [review]
Two inquiries in the code, @amaredeus 👍

File: src/pages/claim/[code].tsx (hunk 2 above)
Comment (amaredeus) [reply src/pages/claim/[code].tsx:87]: yeah this is the logic that checks if the tweet is already posted. Will revisit this upon merging with reece/cornbread merge. Let me add a TODO comment

**Comment by amaredeus** [comment]
@nonsensecodes PR updated according to comments!

**Review by nonsensecodes — APPROVED** [review]
Awesome! Thank you for updating this, @amaredeus!

**Pull request merged**